src/replication.h

```diff
@@ -166,7 +168,7 @@ int64_t perform_read_write_on_fd(int fd, uint8_t *data, uint64_t len,
 int initialize_volume(spec_t *spec, int, int);
 
 /* Replica default timeout is 200 seconds */
-#define	REPLICA_DEFAULT_TIMEOUT	200
+#define	REPLICA_DEFAULT_TIMEOUT	1
```

Comment:
lets not change this.. let us make a new variable, and change it from test code

Reply:
Ok